storybook/stories/API/chart/Sankey.stories.tsx

```diff
@@ -18,6 +18,7 @@ export default {
     margin,
     data,
     sort: { description: 'Whether to sort the data or not' },
+    wrapperAriaLabel: { description: 'Optional aria-label applied to the wrapper <div> element of the chart' },
```

Comment:
why not just just use native `aria-label` and pass it down instead of adding a new prop?

Reply:
That's a fair point. I admit it's a matter of opinion, but in this case I think having a name that explicitly tells the user what's going to happen with the prop is better. 

What's more, using the native HTML attributes could lead to confusion in another way. If I didn't know better, I'd assume that any "extra" attributes applied to `<Sankey>` would end up on the `<svg>` element, not the wrapper.

Reply:
Hmm yeah I guess makes sense. We really, before making changes like these need to figure out broader accessibility so there isn't opportunity for this confusion in the first place. Imo this should just be normal `ariaLabel` and the user should be able to assume with confidence that it goes to the correct place regardless.